nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/resources/skin/views/doc/paramtypes/paging.ftl

```diff
@@ -12,10 +12,17 @@
   "dataType": "integer",
   "required": false
 },
+{
+  "paramType": "query",
```

Comment:
The is already a param `offset` just before, won't it be a problem?

Reply:
I missed it thanks.